transcription/transcription.go

```diff
@@ -29,3 +32,32 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// DownloadFileFromURL locally downloads an audio file stored at url.
+func DownloadFileFromURL(url string) error {
+	// Taken from https://github.com/thbar/golang-playground/blob/master/download-files.go
+	tokens := strings.Split(url, "/")
+	fileName := tokens[len(tokens)-1]
+
+	// Create the file
```

Comment:
> So @anoshaminai <https://github.com/anoshaminai> commented on this,
asking why it was unnecessary if every other section of code had
overarching comment about its purpose.

I think this code is pretty understandable without comments.

On Friday, March 4, 2016, Yoni Nachmany <notifications@github.com> wrote:

> In transcription/transcription.go
> <https://github.com/hack4impact/audio-transcription-service/pull/24#discussion_r55069975>
> :
>
> > @@ -29,3 +32,32 @@ func msgHeaders(from string, to []string, subject string) string {
> >  	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
> >  	return strings.Join(msgHeaders, "\r\n")
> >  }
> > +
> > +// DownloadFileFromURL locally downloads an audio file stored at url.
> > +func DownloadFileFromURL(url string) error {
> > +	// Taken from https://github.com/thbar/golang-playground/blob/master/download-files.go
> > +	tokens := strings.Split(url, "/")
> > +	fileName := tokens[len(tokens)-1]
> > +
> > +	// Create the file
>
> So @anoshaminai <https://github.com/anoshaminai> commented on this,
> asking why it was unnecessary if every other section of code had
> overarching comment about its purpose.
>
> She recommended changing comment to // Get file contents. I thought I
> pushed this, I guess not -- would that comment still be unneccessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hack4impact/audio-transcription-service/pull/24/files#r55069975>
> .
>


-- 
Benjamin Sandler
University of Pennsylvania | Class of 2018
Candidate for BS in Computer Science & BA in Cognitive Science
954.478.6689 | ben1sandler@gmail.com